packages/manager/apps/hub-next/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@ovh-ux/manager-hub-next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "serve": "vue-cli-service serve",
+    "test:unit": "vue-cli-service test:unit"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^3.0.1",
+    "@ovh-ux/manager-core": "^12.0.0",
```

Comment:
`@ovh-ux/manager-core` requires a peerDep to `angular: ^1.7.x`, why do we need it here ?

Reply:
We don't, i imported it to use "bootstrapApplication" but we don't need it anymore, i'll remove it, thanks !